packages/jellyfish-api-core/__tests__/category/spv/decodeHtlcScript.test.ts

```diff
@@ -0,0 +1,121 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { RpcApiError } from '@defichain/jellyfish-api-core'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+
+describe('Spv', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  /** Util function to replace a char at index in string */
+  const replaceAt = (s: string, index: number, replacement: string): string => s.substr(0, index) + replacement + s.substr(index + replacement.length)
+
+  it('should createHtlc', async () => {
+    const pubKeyA = await container.call('spv_getaddresspubkey', [await container.call('spv_getnewaddress')])
+    const pubKeyB = await container.call('spv_getaddresspubkey', [await container.call('spv_getnewaddress')])
+    const blocks = 10
+    const htlc = await container.call('spv_createhtlc', [pubKeyA, pubKeyB, `${blocks}`])
```

Comment:
```suggestion
    const htlc = await container.call('spv_createhtlc', [pubKeyA, pubKeyB, blocks])
```

Reply:
Getting `DeFiDRpcError: 'JSON value is not a string as expected', code: -1` this way